log/development.log

```diff
@@ -2019,3 +2019,198 @@ Processing by Users::RegistrationsController#create as HTML
 Completed 200 OK in 198ms (Views: 26.6ms | ActiveRecord: 0.8ms)
 
 
+Started GET "/" for ::1 at 2019-04-01 11:21:40 +0100
```

Comment:
can we add development.log to .gitignore?

Reply:
it's already in gitignore, but think it's still adding because it hasn't been removed (since it was added), I can do that with this PR.

Reply:
ok, that would be great